workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/ActionNode.java

```diff
@@ -163,8 +155,6 @@ public void setProcessInfo(ProcessInfo processInfo) {
             containingNodes.add(fileRegistrationNode);
         } else if (processInfo.getProcessTypeId() == SEMANTIC_ACTION) {
 
-        } else if (processInfo.getProcessTypeId() == INGESTION) {
-
         } else if (processInfo.getProcessTypeId() == EXPORT_ACTION) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00108)